giraffe/src/components/Axes.tsx

```diff
@@ -100,55 +99,70 @@ export const drawAxes = ({
   context.textAlign = 'end'
   context.textBaseline = 'middle'
   const yDomainWidth = yDomain[1] - yDomain[0]
+  let count = 0
+  for (const yTick of yTicks) {
+    let y
+    if (typeof yTick === 'number') {
+      y = yScale(yTick) + margins.top
+      if (
+        Math.abs(y - margins.top) > GRID_LINE_MIN_DIST &&
+        Math.abs(y - (height - margins.bottom)) > GRID_LINE_MIN_DIST
+      ) {
+        context.strokeStyle = gridColor
+        context.globalAlpha = gridOpacity
+        context.beginPath()
+        context.moveTo(margins.left, y)
+        context.lineTo(width - margins.right, y)
+        context.stroke()
+      }
 
-  if (yColumnType === 'string') {
-    //TODO: implement
-  } else {
-    for (const yTick of yTicks) {
-      let y
-      if (typeof yTick === 'number') {
-        y = yScale(yTick) + margins.top
-        if (
-          Math.abs(y - margins.top) > GRID_LINE_MIN_DIST &&
-          Math.abs(y - (height - margins.bottom)) > GRID_LINE_MIN_DIST
-        ) {
-          context.strokeStyle = gridColor
-          context.globalAlpha = gridOpacity
-          context.beginPath()
-          context.moveTo(margins.left, y)
-          context.lineTo(width - margins.right, y)
-          context.stroke()
-        }
+      context.globalAlpha = 1
+      context.fillStyle = tickFontColor
+      context.fillText(
+        yTickFormatter(yTick, {domainWidth: yDomainWidth}),
+        margins.left - TICK_PADDING_RIGHT,
+        y
+      )
+    } else if (typeof yTick === 'string') {
+      y = yScale(count) + margins.top - height / (yTicks.length * 2)
+      context.globalAlpha = 1
+      context.fillStyle = tickFontColor
 
-        context.globalAlpha = 1
-        context.fillStyle = tickFontColor
-        context.fillText(
-          yTickFormatter(yTick, {domainWidth: yDomainWidth}),
-          margins.left - TICK_PADDING_RIGHT,
-          y
-        )
-      } else {
-        y = yTick
-        if (
-          Math.abs(y - margins.top) > GRID_LINE_MIN_DIST &&
-          Math.abs(y - (height - margins.bottom)) > GRID_LINE_MIN_DIST
-        ) {
-          context.strokeStyle = gridColor
-          context.globalAlpha = gridOpacity
-          context.beginPath()
-          context.moveTo(margins.left, y)
-          context.lineTo(width - margins.right, y)
-          context.stroke()
-        }
+      context.fillText(
+        yTickFormatter(yTick, {domainWidth: yDomainWidth}),
+        margins.left - TICK_PADDING_RIGHT,
+        y
+      )
+      count += 1
+    } else {
```

Comment:
could yTick be something other than number or string?

Reply:
I don't believe so, since time is handled as a number

Reply:
so this `else` existed to handle when there was a string appended to the number, like a number with a label, right?

does the `else if (typeof yTick === 'string')` conditional cover all the logic down here? i.e. have you tested it with a number that has a label? it looks like this `else`, which the number with label would hit, isn't doing the same things as the `else if (typeof yTick === 'string')` above and so I wouldn't expect the storybook to act the same.

if `yTick` could only be a number or a string, we need to remove one of these `else` conditionals.